lib/src/scss-shared/_nx-text-helpers.scss

```diff
@@ -7,7 +7,8 @@
 @import 'nx-variables';
 
 @mixin nx-font-family() {
-  font-family: Proxima Nova, Helvetica Neue, Helvetica, sans-serif;
+  @import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;600;700&display=swap');
```

Comment:
Cannot have external URL references in our products

Reply:
> Cannot have external URL references in our products

This required a little bit more rework than I expected: https://github.com/sonatype/sonatype-react-shared-components/pull/4/commits/26df72ac46e83bcc98402b7b2e23b4257be1ee84